packages/client/src/components/interface/Navigation.tsx

```diff
@@ -102,7 +108,11 @@ export const WORKQUEUE_TABS = {
 
 const GROUP_ID = {
   declarationGroup: 'declarationGroup',
-  menuGroup: 'menuGroup'
+  menuGroup: 'menuGroup',
+  leaderboards: 'leaderboards',
+  statistics: 'statistics',
+  report: 'report',
+  dashboard: 'dashboard'
```

Comment:
Is this correct? I would expect to see

`analytics: 'analytics'` here as there's only one navigation group we're adding 🤔 

>All of the items should only be available for National Sys Admins, National Registrars, Performance Managers. Registrars & registration agents should only see “Performance” link.

According to this all users mentioned here should have access to see the group, but only national sys admin, national registrar and perf manager can see the 3 first items